src/policy/concrete.rs

```diff
@@ -327,7 +327,7 @@ impl<Pk: MiniscriptKey> Policy<Pk> {
     /// combination of timelocks and heightlocks
     pub fn is_valid(&self) -> Result<(), PolicyError> {
         self.check_timelocks()?;
-        self.check_duplicate_keys()?;
+        // self.check_duplicate_keys()?;
```

Comment:
Is there any specific reason for this requirement to be removed? 
This mainly concerns some algorithms the Tapscript Compiler #291 and #342 which might require changes.